tools/generate-integration-docs-screenshot

```diff
@@ -203,13 +226,20 @@ def capture_last_message_screenshot(bot: UserProfile, image_path: str) -> None:
 
 
 def generate_screenshot_from_config(
-    integration_name: str, screenshot_config: ScreenshotConfig
+    integration_name: str, screenshot_config: BaseScreenshotConfig
 ) -> None:
     integration = get_integration(integration_name)
     fixture_path, image_path = get_fixture_and_image_paths(integration, screenshot_config)
     bot = create_integration_bot(integration, screenshot_config.bot_name)
     create_integration_stream(integration, bot)
-    message_sent = send_bot_payload_message(bot, integration, fixture_path, screenshot_config)
+    if isinstance(integration, WebhookIntegration):
+        assert isinstance(
+            screenshot_config, ScreenshotConfig
+        ), "Webhook integrations reqruie ScreenshotConfig"
+        message_sent = send_bot_payload_message(bot, integration, fixture_path, screenshot_config)
+    else:
+        send_bot_mock_message(bot, integration, fixture_path, screenshot_config)
```

Comment:
Should this have some sort of equivalent assertion to the `ScreenshotConfig` one above?

Reply:
Not really, because we expect `BaseScreenshotConfig` for this one, which is specified in the function header already.

Reply:
OK, makes sense.